rotkehlchen/exchanges/bitmex.py

```diff
@@ -129,11 +137,30 @@ def edit_exchange_credentials(self, credentials: ExchangeAuthCredentials) -> boo
         return changed
 
     def first_connection(self) -> None:
+        if self.first_connection_made is True:
+            return
+        try:
+            self.get_scales()
+        except RemoteError as e:
+            log.error(f'Failed to obtain bitmex decimals due to {e}. First connection not made')
+            return
+
         self.first_connection_made = True
 
+    def get_scales(self) -> None:
```

Comment:
Scales? Can you add a docstring?

Reply:
It is what we call decimals for tokens/assets. Updated in the docstring